flag_initialization/default_flag_settings.json

```diff
@@ -3,5 +3,7 @@
   "addressComplete": false,
   "scheduleClosingDate": false,
   "apiAccess": false,
-  "saveAndResume": false
+  "saveAndResume": false,
+  "hCaptcha": false,
+  "formTimer": false
```

Comment:
We could also default this to `true` and leave only hCaptcha `false` to start. The important thing is to especially remember to enable the formTimer if hCaptcha is disabled.